make installation scripts readable at completion

PalashBansal opened the pull request:
If running scripts by double click instead of manually opening cmd- On completion of installation and demo scripts, cmd closes automatically, preventing user from seeing the installation status(whether succeeded or failed at any step)
After the added instructions- both scripts at the completion looks like below screenshots-
![image](https://user-images.githubusercontent.com/6822117/67431741-8e57d880-f602-11e9-84d0-b2b0dc25014a.png)
![image](https://user-images.githubusercontent.com/6822117/67431757-944db980-f602-11e9-9457-4d2ad7e5f188.png)

PalashBansal commented:
Or i guess, its because there is no user input in appveyor build after-
"SUCCESS! Installation is completed"
"Press any key to exit..."
So, its just waiting till max time- 60min and showing failure.
@ihhub Can you please look into this and suggest what's exactly the cause of failure.

ihhub commented:
Hi @PalashBansal , you are correct, AppVeyor waits user input with 30 minutes as time limit for one job run. A wise solution would be to add these lines of code into script during deployment (making an archive for GitHub releases). Are you familiar with this?

PalashBansal commented:
> add these lines of code into script during deployment (making an archive for GitHub releases). Are you familiar with this?

I am really sorry but i don't know about this. Will all next Appveyor build fail if this change is included?
If **not**, I believe as we know the cause of Appveyor auto-build failure, we can ignore this failure.
If **yes**, then how will any new user be helped, because they fork-clone repo like i did which contains scripts(not having pause) causing the trouble. Also, is there any way to give a key input through appveyor.yml file.

PalashBansal commented:
> Hi @PalashBansal , you are correct, AppVeyor waits user input with 30 minutes as time limit for one job run. A wise solution would be to add these lines of code into script during deployment (making an archive for GitHub releases). Are you familiar with this?

@ihhub  I am assuming that this solution will make 2 separate scripts, one for github repo and other for deployed copy. But still, issue will be there for people forking repo. Please confirm if my understanding is correct?
I updated this PR with another fix-
As appveyor environment variables can be seen using- https://github.com/ihhub/fheroes2/pull/200/file
https://ci.appveyor.com/project/ihhub/fheroes2/builds/28386537/job/q2lk77j8bu7dadv4 shows APPVEYOR_REPO_PROVIDER=gitHub which is accessible in batch script, so i updated it with a check.
Now, this change won't effect Appveyor build which has already passed now, but will pause cmd in end user's machine.

ihhub commented:
Hi @PalashBansal , thank you very much for the contribution to the project!
